jest-preset-angular giving error after migration to latest

### Current Behavior

npm ERR! Found: @angular-devkit/build-angular@16.0.0
npm ERR! node_modules/@angular-devkit/build-angular
npm ERR!   dev @angular-devkit/build-angular@"16.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular-devkit/build-angular@">=13.0.0 <16.0.0" from jest-preset-angular@13.0.1
npm ERR! node_modules/jest-preset-angular
npm ERR!   dev jest-preset-angular@"13.0.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.


### Expected Behavior

npm install should install all dependencies correctly

### GitHub Repo

_No response_

### Steps to Reproduce

1. nx migrate latest
2. npm install


### Nx Report

```shell
NA
```


### Failure Logs

_No response_

### Operating System

- [ ] macOS
- [ ] Linux
- [ ] Windows
- [ ] Other (Please specify)

### Additional Information

_No response_

--- a/packages/angular/migrations.json
+++ b/packages/angular/migrations.json
@@ -1110,6 +1110,22 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "16.1.3-jest": {
+      "version": "16.1.3-beta.0",
+      "requires": {
+        "@angular-devkit/build-angular": ">=13.0.0 <17.0.0",
+        "@angular/compiler-cli": ">=13.0.0 <17.0.0",
+        "@angular/core": ">=13.0.0 <17.0.0",
+        "@angular/platform-browser-dynamic": ">=13.0.0 <17.0.0",
+        "jest": "^29.0.0"
+      },
+      "packages": {
+        "jest-preset-angular": {
+          "version": "~13.1.0",
+          "alwaysAddToPackageJson": false
+        }
+      }
     }
   }
 }

--- a/packages/angular/src/utils/versions.ts
+++ b/packages/angular/src/utils/versions.ts
@@ -25,6 +25,6 @@ export const postcssUrlVersion = '~10.1.3';
 export const autoprefixerVersion = '^10.4.0';
 export const tsNodeVersion = '10.9.1';
 
-export const jestPresetAngularVersion = '~13.0.0';
+export const jestPresetAngularVersion = '~13.1.0';
 export const typesNodeVersion = '16.11.7';
 export const jasmineMarblesVersion = '^0.9.2';

